mne_bids/mne_bids.py

```diff
@@ -386,10 +457,20 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, eeg_ref=None,
         location of the reference electrode.  Defaults to None.
     eeg_gnd : str
         Description  of the location of the ground electrode. Defaults to None.
+    write_mode : str, one of ('overwrite', 'error')
+        How the file should be handled if is exists already.
+        Defaults to 'overwrite'.
+        If write_mode == `overwrite` the old file will be removed and replaced
+        with a new one.
+        If write_mode == `error` an `OSError` will be raised.
     verbose : bool
         Set verbose output to true or false. Defaults to true.
 
     """
+    if op.exists(fname) and write_mode == 'error':
+        raise OSError(EEXIST, '"%s" already exists. Please set'
+                      ' write_mode to "overwrite" or "append".' % fname)
```

Comment:
can you refactor this into a small private function `_check_file_exists` or something like that? It will also make it easier to test.